basilisk/multiprocessing_worker.py

```
@@ -0,0 +1,93 @@
+	task: Callable, result_queue: Queue, cancel_flag, *args: Any, **kwargs: Any
+) -> None:
+	"""Run a task in a separate process.
+
+	Args:
+		task: The task to run
+		result_queue: The queue to store the task result
+		cancel_flag: The flag to indicate if the task should be cancelled
+		*args: The task arguments
+		**kwargs: The task keyword arguments
+	"""
+	try:
```

Pylint (3.3.7)

[warning] Bad indentation. Found 1 spaces, expected 4

(W0311)